server/models/ipfspins.ts

```diff
@@ -0,0 +1,41 @@
+import * as Sequelize from 'sequelize';
+import { Model, DataTypes } from 'sequelize';
+import { ModelStatic, ModelInstance } from './types';
+
+export type IpfsPinsAttributes = {
+  id: number;
+  ipfs_hash: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+export type IpfsPinsInstance = ModelInstance<IpfsPinsAttributes>;
+
+export type IpfsPinsModelStatic = ModelStatic<IpfsPinsInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes,
+): IpfsPinsModelStatic => {
+  const IpfsPins = <IpfsPinsModelStatic>sequelize.define('IpfsPins', {
+    id: { type: dataTypes.INTEGER, primaryKey: true },
+    ipfs_hash: { type: dataTypes.STRING, allowNull: false },
+  }, {
```

Comment:
we should also store the `address_id` of the user's active address when submitting the data to be pinned to IPFS

Reply:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/11271352/157310414-24258df0-200a-4f8f-b104-b6212299edae.png">
Fixed this

Reply:
it's not in the model defined here

Reply:
I'm not sure why you don't see it. Here it is in line number 7 &25
<img width="631" alt="image" src="https://user-images.githubusercontent.com/11271352/157383676-eff88b59-d952-4f6e-8332-b29805fee220.png">